client/db/bolt/upgrades.go

```diff
@@ -115,8 +101,7 @@ func getVersionTx(tx *bbolt.Tx) (uint32, error) {
 	}
 	versionB := bucket.Get(versionKey)
 	if versionB == nil {
-		// A nil version indicates a version 0 database.
-		return 0, nil
+		return 0, fmt.Errorf("database version not found")
```

Comment:
We set the DB to version 1 with the merging of #448 on July 6th, 2020, long before we tagged release-v0.1.0 on Oct 20, 2020, so this is fine.